src/persistence/settings.cpp

```diff
@@ -223,6 +223,7 @@ void Settings::loadGlobal()
             else
                 style = "None";
         }
+		avatarsPath = s.value("avatarsPath",QDir::homePath()).toString();
```

Comment:
This will always save QDir::homePath in settings.ini... I think that's not what you want, otherwise this PR is useless?